lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -66,6 +66,17 @@
   .form-actions {
     padding-top: gutter();
   }
+
+  &.-signup {
+    width: 30%;
+    min-width: 256px;
+    margin: 1.5%;
+    display: inline-block;
+
+    .beta-card__row:first-of-type {
+      margin-top: 0;
+    }
+  }
```

Comment:
Hmmm, not keen on this class. Maybe we change the base card to add that top margin/padding on the heading and take it off the card. This way, if there is a heading then the right spacing is added, otherwise none is.